.pre-commit-config.yaml

```
@@ -57,5 +57,10 @@ repos:
         args: [--prose-wrap=always, --print-width=88]
         exclude: ^.github/ISSUE_TEMPLATE/bug_report.md$
 
+  - repo: https://github.com/pre-commit/mirrors-mypy
+    rev: "v0.982"
+    hooks:
+      - id: mypy
```

Let's add these arguments to make the errors a bit more helpful, and also exclude the tests to narrow the scope:
```suggestion
      - id: mypy
        args: [--pretty, --show-error-codes]
        exclude: ^tests/      
```